dask/array/backends.py

```diff
@@ -108,8 +116,8 @@ def _tensordot(a, b, axes=2):
 def register_cupy():
     import cupy
 
+    from dask.array.dispatch import percentile_dispatch
     from dask.dataframe.backends import percentile
```

Comment:
This line will cause us to try to `import pandas`. Should we update this to avoid importing from `dask.dataframe`?